src/python/bcc/table.py

```diff
@@ -403,13 +404,123 @@ def close_perf_buffer(self, key):
             del(self.bpf.open_kprobes()[(id(self), key)])
         del self._cbs[key]
 
-class PerCpuHashTable(TableBase):
+class PerCpuHash(HashTable):
     def __init__(self, *args, **kwargs):
-        raise Exception("Unsupported")
+        self.reducer = kwargs["reducer"]
```

Comment:
You should probably use `kwargs.pop("reducer", None)` to avoid KeyError as well as combine this and the line below into a single statement.